token/confidential-transfer/proof-extraction/src/encryption.rs

```diff
@@ -51,6 +51,36 @@ impl PodFeeCiphertext {
 #[repr(C)]
 pub struct PodBurnAmountCiphertext(pub(crate) PodGroupedElGamalCiphertext3Handles);
 
+impl PodBurnAmountCiphertext {
+    pub fn extract_commitment(&self) -> PodPedersenCommitment {
+        self.0.extract_commitment()
+    }
```

Comment:
Actually it looks like it's never used on the others either. @samkim-crypto are these ever needed?

Reply:
Yeah this function is not currently used anywhere in the repo currently. This is a remnant from a proof logic that lived in `program-2022`, which has since been replaced by a proof extraction function from zk-sdk. It is natural function to have, but let's remove it here for now.